easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.4rc1-GCC-11.3.0.eb

```diff
@@ -0,0 +1,32 @@
+name = 'OpenMPI'
+version = '4.1.4rc1'
+
+homepage = 'https://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['46169edd9cf8366608684e5c6a342650b2393d46c9f9a380add8e09f1f8001de']
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('zlib', '1.2.12'),
+    ('hwloc', '2.7.1'),
+    ('libevent', '2.1.12'),
+    ('UCX', '1.12.1'),
```

Comment:
we should add UCC as well in the final Open MPI
https://github.com/openucx/ucc/

Reply:
I imagine that'll need some patching though, since it can be built with/without CUDA support?